README.md

```diff
@@ -65,3 +65,37 @@ The notes database and settings file are stored in:
 **Windows** : `C:\Users\user\AppData\Roaming\Awesomeness`  
 **Linux** &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: `/home/user/.config/Awesomeness/` **or** `/home/snap/notes/x1/.config/Awesomeness` **(using snap)**  
 **Mac** &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: `/home/user/.config/Awesomeness/`
+
+## Contributers
+
+### Developers:
+Thorbjørn Lindeijer  
```

Comment:
I don't think I deserve to be listed first because my contributions haven't been that many. The current list does not look sorted in any particular way. I would suggest either using commit count (though it is only a rough approximation of contribution size) or alphabetical. :-)

Reply:
I agree with alphabetical sorting, I definitely shouldn't be third place when I only made two big contributions (qt6 and cmake). Looking at the [contributors graph](https://github.com/nuttyartist/notes/graphs/contributors), that would be unfair to the other contributors.

Reply:
Alphabetical sorting it is 👍